includes/editor-templates/templates.php

```diff
@@ -126,11 +126,11 @@
 					</div>
 					<span class="divider"></span>
 					<div id="elementor-template-library-view-grid" class="elementor-template-library-action-item">
-						<i class="eicon-container-grid" aria-hidden="true" title="Grid view"></i>
+						<i class="eicon-library-grid" aria-hidden="true" title="Grid view"></i>
```

Comment:
@max-zu Please fix the translations in the `title` attributes. Looks like we missed it. In the JS files as well. Thanks!